src/main/java/exception/CustomException.java

```diff
@@ -0,0 +1,11 @@
+package exception;
+
+public class CustomException extends RuntimeException {
```

Comment:
I did not mean literally "Custorm" 😅 
we should have a proper name for it, depending on the context of the place where it is thrown, and 1 exception most likely won't be enough, because we have a big application here with lots of different logic, and each case should be considered on it's own, for example:
`InvalidInputDataException`, `NotEnoghFruitsException`, or some places can reuse the ones from jdk, like `IllegalArgumentException`, Overall it is fine to reuse exceptions but only if they are fitting in the context, and they should be named properly